lib/templates/404error.html

```diff
@@ -0,0 +1,20 @@
+{{>head}}
+
+  <div class="col-xs-6 col-sm-3 col-md-2 sidebar sidebar-offcanvas-left">
+    <h5><span class="package-name">{{self.name}}</span> <span class="package-kind">{{self.kind}}</span></h5>
+    {{>packages}}
+  </div>
+
+  <div class="col-xs-12 col-sm-9 col-md-8 main-content">
+    <h1>404: Something's gone wrong :-(</h1>
+
+    <section class="desc">
+      <p>You've stumbled onto a page that doesn't exist.  Luckily this site
+         has other <a href="index.html">pages</a>.</p>
+      <p>If you were looking for something specific, maybe try the search
+         field located in the upper right corner, or the index located on
```

Comment:
located in the upper right corner -> (at the upper right)
, or -> or
index located on the left -> library index (at the left)